CP02PMCO/CP02PMCO_D00011_ingest.csv

```diff
@@ -1,5 +1,5 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00011/imm/adcpt/adcpt*.DAT,CP02PMCO-RII01-02-ADCPTG010,telemetered,Available,no data?
+#mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCO/D00011/imm/adcpt/adcpt*.DAT,CP02PMCO-RII01-02-ADCPTG010,telemetered,Not available,Inductive comms failure; no telemetered data

```

Comment:
Confirmed that these data were not available. Commented out the line, changed the status and added a comment